config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-v1beta1.yaml

```
@@ -372,3 +372,40 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: capz-pr-conformance-k8s-ci-v1beta1
+  - name: pull-cluster-api-provider-azure-e2e-workload-upgrade-v1beta1
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 4h
+    branches:
+    - ^release-1.*
+    path_alias: sigs.k8s.io/cluster-api-provider-azure
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220624-1a63fdd9f2-1.23
```

should be the latest 1.24 image now

```suggestion
      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220722-16ae0286c2-1.24
```

---

After rebasing on top of latest master it in fact seems that `gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220726-61b7bafca6-1.24` is the newest image we want. Thanks for pointing out that we're no longer using 1.23!